pandas/core/tools/datetimes.py

```diff
@@ -200,19 +200,31 @@ def _convert_listlike_datetimes(arg, box, format, name=None, tz=None,
         if format is not None:
             raise ValueError("cannot specify both format and unit")
         arg = getattr(arg, 'values', arg)
+        tz_parsed = None
         result = tslib.array_with_unit_to_datetime(arg, unit,
                                                    errors=errors)
+        # GH 25546:
+        # Capture tz information from array_with_unit_to_datetime
```

Comment:
this is not great; should just change all returns of array_with_unit_to_datetime so that a tuple is returned; otherwise this is *very* confusing. pls update that doc-string as well.